### ISSUE TITLE
Add all available languages to the languages bar

### ISSUE THREAD

**Issue by linderr** [findthemasks/findthemasks#682]
The new header has a languages bar. 
Right now it includes these 8 languages.
![image](https://user-images.githubusercontent.com/62862914/80151628-3c6f1b80-856f-11ea-89d6-28b1c4a84dae.png)

According to Pull Requests, we also have: Arabic, Hindi, Japanese, Korean, and Russian
![image](https://user-images.githubusercontent.com/62862914/80152084-e64ea800-856f-11ea-84cc-dbcb55bb9198.png)

For a total of 13 languages. Let's actually allow the public to do that :)

**Comment by nburt** [comment]
As a big caveat many of these languages have not been proofread. Also Arabic will need special attention since it's a right to left language. No idea if our layout will make sense with it or not.

**Comment by linderr** [comment]
Ohh fair enough. Do we need more translators for these languages?? I can definitely recruit

**Comment by nburt** [comment]
yes, I also want to make sure that we actually need these languages. We don't support Japan, Korea, Russia right now, do we actually need them?

**Comment by linderr** [comment]
Need is hard to define. I think it's a chicken/egg problem.
Many people are bilingual to some degree, and some areas like Japan where people don't really speak English at all, FTM would be daunting --- a visually intuitive tool but not for them. If we offered JP support, they could see it and then ask some dev friends to join (lowkey the long term strategy i'm going for right now by recruiting Yoshi).

Also older people don't bother to learn English in some cultures. My grandparents have been in Canada for 30 years and don't speak a lick of English. They would be happy to donate or do something but wouldn't be accessible to the site without CH.

**Comment by linderr** [comment]
Removed from milestone! I'm in it for the long con :)
